src/types.rs

```diff
@@ -24,9 +25,35 @@ use crate::{
 };
 
 /// A buffer of bytes.
-#[derive(Clone, PartialEq, Eq, Size, FromBytesWithInput, ToBytes)]
+#[derive(Clone, PartialEq, Eq, Size)]
 pub struct Buffer(#[neli(input)] Vec<u8>);
```

Comment:
If you're overriding the derive implementations, please remove the inner attribute meant for those macros.